backend/start/routes.ts

```diff
@@ -84,3 +85,7 @@ router.post("user/login", [AuthController, "store"]).use(middleware.guest());
 router
   .delete("user/logout", [AuthController, "destroy"])
   .use(middleware.usosAuth());
+
+router
+  .post("/user/toggleNotifications", [UsersController, "toggleNotifications"])
+  .use(middleware.usosAuth());
```

Comment:
to nie jest restowe API, raczej to powinien być endpoint typu

.patch(`/user`) gdzie wysyłasz co chcesz zmienić w ustawieniach usera

Reply:
okej, ale w sumie my chcemy zmieniać tylko allowNotifications z `alowNotifications = !allowNotifications`, więc body może być puste? legit ten pomysł jest?

Reply:
raczej powinienieś wysłać { allowNotification: true | false } z frontendu, łatwiej w przyszłości będzie dorzucić kolejne pola

Reply:
https://stackoverflow.com/a/5030057

Reply:
w tym pytaniu możesz więcej sobie przeczytać jak coś takiego można zaprojektować

Reply:
o spoko dzieki